### ISSUE TITLE
How to set Timezone from the Kameleo API?

### ISSUE THREAD

**Issue by JonathanR25** [kameleo-io/local-api-client-typescript#12]
What would be the proper code syntax form to set the Timezone for a profile from the Kameleo API?

Also, could your provide some example code?

**Comment by MBailey01** [comment]
Here is an example that you can use to set the Timezone for a profile through the API:

```
const createProfileRequest = BuilderForCreateProfile
  .forBaseProfile(BaseProfileList[0].id)
  .setTimezone({
    value: 'manual',
    extra: 'America/New_York'
  })
  .build();
const profile = await client.createProfile({ body: createProfileRequest });
```

These answers are provided from the Kameleo [Set Timezone from API](https://help.kameleo.io/hc/en-us/articles/13603071379740-Set-Timezone-from-APII) article.

**Comment by KameleoStaff** [comment]
We recently released Kameleo v4.0.0, which introduced several changes and improvements to the platform. As a result, this code has been updated to be fully compatible with the latest version.

```
@type {import('@kameleo/local-api-client').CreateProfileRequest} */
const createProfileRequest = {
  fingerprintId: fingerprints[0].id,
  timezone: {
    value: 'manual',
    extra: 'America/New_York'
  }
};

const profile = await client.profile.createProfile(createProfileRequest);
```

These answers are provided from the Kameleo [Set Timezone from API](https://help.kameleo.io/hc/en-us/articles/13603071379740-Set-Timezone-from-API) article.
